### ISSUE TITLE
bug: crash after selecting "Nothing in here, all clear, nice job!" and then pressing down key

### ISSUE THREAD

**Issue by olivercalder** [daveshanley/vacuum#292]
I'm getting a crash when I select the "Nothing in here, all clear, nice job!" entry from within the dashboard (by pressing Enter), and then pressing Down as if to select the first problem (of which there are none). I'm running the most recent release available via the "Install using curl" option as of 2023-06-12.

The error message is as follows:
```
 INFO  Linting against 42 rules: https://quobix.com/vacuum/rulesets/recommended
panic: runtime error: index out of range [-1]

goroutine 1 [running]:
github.com/gizak/termui/v3/widgets.(*List).Draw(0xc0004a0120, 0xc0003ba360)
	/home/runner/go/pkg/mod/github.com/gizak/termui/v3@v3.1.0/widgets/list.go:47 +0x69c
github.com/gizak/termui/v3.(*Grid).Draw(0xc0005fc9c0, 0xc0003ba330?)
	/home/runner/go/pkg/mod/github.com/gizak/termui/v3@v3.1.0/grid.go:157 +0x17e
github.com/gizak/termui/v3.Render({0xc000713828?, 0x2, 0x2?})
	/home/runner/go/pkg/mod/github.com/gizak/termui/v3@v3.1.0/render.go:25 +0x179
github.com/daveshanley/vacuum/cui.(*Dashboard).eventLoop(0xc000002300, {0xc000622140?, 0x8, 0x80?})
	/home/runner/work/vacuum/vacuum/cui/dashboard.go:190 +0x652
github.com/daveshanley/vacuum/cui.(*Dashboard).Render(0xc000002300)
	/home/runner/work/vacuum/vacuum/cui/dashboard.go:122 +0xa3d
github.com/daveshanley/vacuum/cmd.GetDashboardCommand.func2(0xc000004f00?, {0xc00039b240, 0x1?, 0x1?})
	/home/runner/work/vacuum/vacuum/cmd/dashboard.go:108 +0x41d
github.com/spf13/cobra.(*Command).execute(0xc000004f00, {0xc00039b200, 0x1, 0x1})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x15874c0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/daveshanley/vacuum/cmd.Execute({0xfc1f04?, 0x0?}, {0xfc8840?, 0x0?}, {0xfc42f0?, 0xc0000061a0?})
	/home/runner/work/vacuum/vacuum/cmd/root.go:44 +0xaa
main.main()
	/home/runner/work/vacuum/vacuum/vacuum.go:22 +0x10e
```

See attached screenrecording for an example of the bug (don't mind the wiggling window, it's to get the screenrecorder to register changes in the window -- no effect on the actual behavior). 
[Screencast from 2023-06-12 11-22-42.webm](https://github.com/daveshanley/vacuum/assets/45443514/6d2c338b-a6b4-4efb-bd05-392347015148)

**Comment by daveshanley** [comment]
Hi, I am glad you found this bug. It's been there since day one - but I didn't want to fix it as a test to see if anyone was using the console. So you won the easter egg hunt!

Now it will be fixed :)

**Comment by daveshanley** [comment]
Fixed and available in `v0.1.6`. Thank you!
